patches/server/0010-Adventure.patch

```diff
@@ -4895,6 +4895,26 @@ index 5540a766bc7edf8985058e29cc419c2c6b1d4428..baff11728120fda1740d5e15163760f5
  
      @Override
      public String getDisplayName() {
+@@ -303,4 +360,19 @@ final class CraftTeam extends CraftScoreboardComponent implements Team {
+         return !(this.team != other.team && (this.team == null || !this.team.equals(other.team)));
+     }
+ 
++    // Paper start - make Team extend ForwardingAudience
++    @Override
```

Comment:
I am not sure but we might want to state, that it doesn't necessarily contain all members of the team as audience.